src/index.html

```diff
@@ -3,11 +3,318 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>BANG & OLUFSEN</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <header class="header" id="home">
+      <nav class="navigation header__navigation">
+        <div class="navigation__logo">
+          <a href="#">
+            <img
+              class="navigation__logo--img"
+              src="images/logo.svg"
+              alt="Logo"
+            >
+          </a>
+        </div>
+
+        <div class="navigation__icons">
+          <a class="phone" href="+1 234 555-55-55">
+            <span class="phone__label">+1 234 555-55-55</span>
+            <img
+              class="phone__icon"
+              src="images/Phone.svg"
+              alt="Phone"
+            >
+          </a>
+
+          <a class="burger" href="#menu">
+            <span class="burger__label">Open menu</span>
+          </a>
+        </div>
+      </nav>
+      <h1 class="title header__title">
+        Your senses.
+        <br>
+        Set free.
+      </h1>
+    </header>
+
+    <div class="menu" id="menu">
+      <nav class="navigation menu__navigation">
+        <div class="navigation__logo">
+          <a href="#">
+            <img
+              class="navigation__logo--img"
+              src="images/logo.svg"
+              alt="Menu"
+            >
+          </a>
+        </div>
+
+        <a href="#home">
+          <img
+            class="close__icon"
+            src="images/Close.svg"
+            alt="Close"
+          >
+        </a>
+      </nav>
+
+      <nav class="links menu__links">
+        <ul class="links__list">
+          <li class="links__item"><a href="#" class="links__link">Speakers</a></li>
```

Comment:
Those should be links to proper sections, make sure the scroll is smooth

Reply:
On page B&O unfortunately are not sections like Speakers, Headphones, Televisions, Soundbars, World of B&O

Reply:
Yup, this must be some kind of an error on mates part, for now you could rename those sections in the menu to more accurately reflect sections in the layout and implement smooth scroll.